### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by jlord**
@maxbrunsfeld here's a readme, lemme know if I've missed something :sparkles:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
 # Line Ending Selector [![Build Status](https://travis-ci.org/atom/line-ending-selector.svg?branch=master)](https://travis-ci.org/atom/line-ending-selector)
+
+![status bar tile](https://cloud.githubusercontent.com/assets/1305617/9274149/6b317568-4293-11e5-83ba-614a6c0d9890.png)
+
+
+This is an [Atom](https://atom.io) package that displays the current line ending type of a file: `CRLF` (Windows), `LF` (Unix), or `Mixed` (both). It also lets the user change the line ending of a file.
```
Comment (maxbrunsfeld) [inline README.md]: I would probably change `lets the user` to `lets you`, since you're addressing the reader directly elsewhere.

**Comment by maxbrunsfeld** [comment]
Very nice! Thanks for writing this up; I think the screenshots pointing it out will be super helpful for people.

File: README.md (hunk 1 above)
Comment (jlord) [inline README.md]: :+1:

**Comment by maxbrunsfeld** [comment]
@jlord I think we should actually remove the section about installing the package, because we're going to bundle it, so it actually becomes a bit of a problem if people install another copy of it into `~/.atom/packages`.

**Comment by jlord** [comment]
Oh yeah, sounds good :+1:

**Comment by maxbrunsfeld** [comment]
:sparkles:

**Pull request merged**